docs/resources/scalr_environment.md

```diff
@@ -23,7 +23,7 @@ resource "scalr_environment" "test" {
 * `name` - (Required) Name of the environment.
 * `account_id` - (Required) ID of the environment account, in the format `acc-<RANDOM STRING>`
 * `cost_estimation_enabled` - (Optional) Set (true/false) to enable/disable cost estimation for the environment. Default `true`.
-* `cloud_credentials` - (Optional) List of the environment cloud-credentials IDs, in the format `cred-<RANDOM STRING>`.
+* `cloud_credentials` - Deprecated. Use `default_provider_configurations` instead.
```

Comment:
@lyzohub how about adding deprecated profile to resource relationship?

Reply:
@soltysss Done! https://github.com/Scalr/fatmouse/pull/5516